ceph-pull-requests-arm64/config/definitions/ceph-pull-requests-arm64.yml

```diff
@@ -0,0 +1,100 @@
+- job:
+    block-downstream: false
+    block-upstream: false
+    builders:
+    - shell: |-
+        bash -ex
+        NPROC=$(nproc)
+        testnproc=$(($NPROC / 4))
+        #export CHECK_MAKEOPTS="-R unittest* -j${testnproc}"
+        export CHECK_MAKEOPTS="-j${testnproc}"
+        export BUILD_MAKEOPTS="-j${NPROC}"
+        ./run-make-check.sh
+        sleep 5
+        ps -ef | grep ceph || true
+    concurrent: true
+    description: <!-- Managed by Jenkins Job Builder -->
+    disabled: false
+    name: !!python/unicode 'ceph-pull-requests-arm64'
+    node: arm64
+    parameters:
+    - string:
+        default: origin/master
+        description: A pull request ID, like 'origin/pr/72/head'
+        name: sha1
+    project-type: freestyle
+    properties:
+    - build-discarder:
+        artifact-days-to-keep: -1
+        artifact-num-to-keep: -1
+        days-to-keep: 15
+        num-to-keep: 300
+    - raw:
+        xml: |
+          <com.sonyericsson.jenkins.plugins.bfa.model.ScannerJobProperty plugin="build-failure-analyzer@1.18.1">
+          <doNotScan>false</doNotScan>
+          </com.sonyericsson.jenkins.plugins.bfa.model.ScannerJobProperty>
+    - github:
+        url: https://github.com/ceph/ceph/
+    - raw:
+        xml: |
+          <com.sonyericsson.rebuild.RebuildSettings plugin="rebuild@1.25">
```

Comment:
why aren't you allowing this to be rebuilt? Or was that just something that trickled down from the UI?